LibreNMS/Alert/Transport/Jira.php

```diff
@@ -1,23 +1,16 @@
 <?php
-/* Copyright (C) 2015 Aldemir Akpinar <aldemir.akpinar@gmail.com>
```

Comment:
Not sure about removing credits on a file like this ... Any reason to do so ?

Reply:
I am not remove any lines or erasing credit from the old transporter, I thew it away and started fresh. 

Besides I don't think there is a line left from the original code that was written 7 years ago especially after murrant rewrote how the whole Transport section works.

Reply:
Fine with me.